WordPress/gutenberg
# [Type] Enhancement - Adds max limit to row span and column span in grid.

## What?
Fixes: #63967 

## Why?
There was no maximum limit set for row span and column span, allowing users to set values larger than the actual number of rows or columns.

## How?
It calculates the max column and row span based on the columns/row count and current position.

## Testing Instructions
1. Adds the grid block.
2. Insert any block into it example image block.
3. Try to increase/decrease the row/column span in the style tab of the block. It should not exceed the max columns/rows.

## Screenshots or screencast 

https://github.com/user-attachments/assets/96e49da5-52d3-47f7-b20b-7e6e01379f47
